src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp

```diff
@@ -116,8 +116,8 @@ class IsotropicHomogeneous : public ConstitutiveRelation<Dim> {
       "respectively. Both are measured in units of stress, typically Pascals."};
 
   IsotropicHomogeneous() = default;
-  IsotropicHomogeneous(const IsotropicHomogeneous&) = delete;
-  IsotropicHomogeneous& operator=(const IsotropicHomogeneous&) = delete;
+  IsotropicHomogeneous(const IsotropicHomogeneous&) = default;
+  IsotropicHomogeneous& operator=(const IsotropicHomogeneous&) = default;
```

Comment:
Could you please add this change to `Test_IsotropicHomogeneous.cpp` so the copy constructor is being tested:

```diff
diff --git a/tests/Unit/PointwiseFunctions/Elasticity/ConstitutiveRelations/Test_IsotropicHomogeneous.cpp b/tests/Unit/PointwiseFunctions/Elasticity/ConstitutiveRelations/Test_IsotropicHomogeneous.cpp
index 1530bd755..871a77b6b 100644
--- a/tests/Unit/PointwiseFunctions/Elasticity/ConstitutiveRelations/Test_IsotropicHomogeneous.cpp
+++ b/tests/Unit/PointwiseFunctions/Elasticity/ConstitutiveRelations/Test_IsotropicHomogeneous.cpp
@@ -34,6 +34,7 @@ void test_type_traits() {
   CHECK(relation !=
         Elasticity::ConstitutiveRelations::IsotropicHomogeneous<Dim>{2., 2.});
   test_serialization(relation);
+  test_copy_semantics(relation);
   const auto created_relation = TestHelpers::test_creation<
       Elasticity::ConstitutiveRelations::IsotropicHomogeneous<Dim>>(
       "BulkModulus: 1.\n"
```